File: Homeworks/C# Part 2/03.Methods/06.FirstLargerThanNeighbours/FirstLargerThanNeighbours.cs
using System;

class FirstLargerThanNeighbours
{
    static int GetFirstNumbersWithSmallerNeighbours(int[] array)
    {
        int index = -1;
        for (int i = 1; i < array.Length - 1; i++)
        {
            if (array[i] > array[i - 1] && array[i] > array[i + 1])
            {
                index = i;
                break;
            }
        }        
        return index;
    }

    static void Main()
    {
        int numberN = int.Parse(Console.ReadLine());
        int[] numbers = new int[numberN];
        string[] input = Console.ReadLine().Split(' ');
        for (int i = 0; i < numberN; i++)
        {
            numbers[i] = int.Parse(input[i]);
        }
        Console.WriteLine(GetFirstNumbersWithSmallerNeighbours(numbers));
    }
}

File: Homeworks/JS Fundamentals/08. Functions/06. First larger than neighbours/first-larger-than-neighbours.js
function solve(args) {
    var numbers = args[1].split(' ');

    function getFirstNumbersWithSmallerNeighbours(array) {
        var index = -1;
        for (var i = 1; i < array.length - 1; i += 1) {
            if (+array[i] > +array[i - 1] && +array[i] > +array[i + 1]) {
                index = i;
                break;
            }
        }
        return index;
    }
    console.log(getFirstNumbersWithSmallerNeighbours(numbers));
}